14/umbraco-cms/fundamentals/data/relations.md

```diff
@@ -34,7 +34,7 @@ To create and use Relations in your Umbraco Website, you need to define a 'Relat
 
 ### Create a Relation Type
```

Comment:
This section needs to be updated at some point. You might see some outdated images here. Currently, the Create button is missing from the UI (open issue in CMS) and hence the section cannot be updated. I've added a task so we can track it later.